### DISCUSSION TITLE
Update links on vue.js

### DISCUSSION THREAD

**Discussion by rhues** [vikejs/vike#1675] [Discussion]
This page still refers to the old vite-plugin-ssr page, and it should be updated to refer to vike.dev.

https://vuejs.org/guide/scaling-up/ssr.html

**Comment by brillout** [comment]
:+1: Let me make a PR.

**Reply by brillout** [reply]
Actually, I think it's best we make a proper PR a little later. Let's first revamp the landing page and finish polishing `vike-vue`. We'll then have a shot to also plug Vike into other sections.
